src/components/autocomplete/demoRepeatMode/index.html

```diff
@@ -0,0 +1,41 @@
+<div ng-controller="DemoCtrl as ctrl" layout="column" ng-cloak>
+  <md-content class="md-padding">
+    <form ng-submit="$event.preventDefault()">
+      <p id="autocompleteDescription">
+        Use the <code>md-mode</code> attribute to fall back to ng-repeat for autocomplete entries.
+        Note that sizable lists can have performance implications for your application. Using modes
+        other than 'virtual' will require you to address those directly within your application.
```

Comment:
This seems to indicate that there may be quite a few modes. Let's just say using standard mode here.

Reply:
Good call :+1: